messenger.go

```diff
@@ -697,3 +740,13 @@ func (m *Messenger) MarkMessagesSeen(ids ...string) error {
 func (m *Messenger) UpdateMessageOutgoingStatus(id, newOutgoingStatus string) error {
 	return m.persistence.UpdateMessageOutgoingStatus(id, newOutgoingStatus)
 }
+
+func Identicon(id string) string {
+	return identicon.GenerateBase64(id)
+}
+
+func GenerateName(id string) string {
```

Comment:
Not used?

Reply:
This one we use in `status-react` (same for identicon), because generation is only in here, we need to export to status-react, as it is used synchronously to generate names (as it was before), otherwise it would require too many changes in one go.